bcrypt.go

```diff
@@ -161,10 +170,11 @@ func HashBytes(password []byte, salt ...[]byte) (hash []byte, err error) {
 	}
 
 	// cipher expects null terminated input (go initializes everything with zero values so this works)
-	password_term := make([]byte, len(password)+1)
-	copy(password_term, password)
+	passwordTerm := make([]byte, len(password)+1)
+	copy(passwordTerm, password)
```

Comment:
This is very nitpicky but, since the entire rest of this module uses snake_case, I wouldn't change it for this one variable either.